src/assets/scss/components/dashboard.scss

```diff
@@ -50,6 +50,10 @@ $sectionAreaSpacing: 15px;
   }
 }
 
+.footerStyling {
```

Comment:
Please use `kebab-case` for CSS class names.

Reply:
My only concern here is that changing to kebab-case isn't consistent with what's been done with the project before. Seems that previous work was done in a mix of camel-case and kebab-case. I've changed my work to use kebab-case, but I think it would also be worth it to go back and make naming conventions consistent, when there's time

Reply:
I think the whole codebase needs to be refactored. There are lots of inconsistencies everywhere. I saw camelCase, PascalCase, kebab-case, snake_cake in the project. Depending on the tech stack, for certain things naming conventions are different depending on that what technology is being used. However, for CSS, no matter what technology, the standard is to either use kebab-case or BEM. So I firmly believe we should start making new code follow the right conventions so it takes less time to refactor later on.